lib/sensors/low_power_bms.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <cstdint>
+#include <memory>
+#include <optional>
+
+#include <core/logger.hpp>
+#include <core/types.hpp>
+#include <io/i2c.hpp>
+
+static constexpr std::uint8_t kDefaultLpBmsAddress  = 0x10;
```

Comment:
this should be in either the `hyped::sensors` namespace or inside `LowPowerBms` if it's only used internally